src/shared/UIElements/Jumbotron/Jumbotron.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <b-jumbotron
+    lead="Bootstrap v4 Components for Vue.js 2"
+    :bg-variant="type.background"
+    :border-variant="type.border"
+    :text-variant="type.text"
+    :header="header"
+  >
+    <slot />
+  </b-jumbotron>
+</template>
+
+<script>
+export default {
+  name: "Jumbotron",
+  props: {
+    type: Object,
+    header: String,
+  },
+};
+</script>
+
+<style></style>
```

Comment:
Why is there an empty `style` tag?

Reply:
I guess just in case? Components should have that option to add style but I left it there so that when you need it you just add